src/main/java/com/myhome/controllers/CommunityController.java

```diff
@@ -20,160 +20,169 @@
+    )
+    public ResponseEntity<GetHouseDetailsResponse> listCommunityHouses(
+            @PathVariable String communityId) {
+        log.trace("Received request to list all houses of community with id[{}]", communityId);
+        var houseDetails = communityService.getCommunityDetailsById(communityId).getHouses();
+        var getHouseDetailsResponseSet =
+                communityApiMapper.communityHouseSetToRestApiResponseCommunityHouseSet(houseDetails);
+
+        var response = new GetHouseDetailsResponse();
+        response.setHouses(getHouseDetailsResponseSet);
+        return ResponseEntity.status(HttpStatus.OK).body(response);
+    }
+
+    @Operation(description = "Add a new admin to the community given a community id")
+    @PostMapping(
+            path = "/communities/{communityId}/admins",
+            produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE},
+            consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE}
+    )
+    public ResponseEntity<AddCommunityAdminResponse> addCommunityAdmin(
+            @PathVariable String communityId, @Valid @RequestBody
+            AddCommunityAdminRequest request) {
+        log.trace("Received request to add admin to community with id[{}]", communityId);
+        var community = communityService.addAdminsToCommunity(communityId, request.getAdmins());
+        var response = new AddCommunityAdminResponse();
+        var adminsSet =
+                community.getAdmins().stream().map(CommunityAdmin::getAdminId).collect(Collectors.toSet());
+        response.setAdmins(adminsSet);
+        return ResponseEntity.status(HttpStatus.CREATED).body(response);
+    }
+
+    @Operation(description = "Add a new house to the community given a community id")
+    @PostMapping(
+            path = "/communities/{communityId}/houses",
+            produces = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE},
+            consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE}
+    )
+    public ResponseEntity<AddCommunityHouseResponse> addCommunityHouse(
+            @PathVariable String communityId, @Valid @RequestBody
+            AddCommunityHouseRequest request) {
+        log.trace("Received request to add house to community with id[{}]", communityId);
+
+        var communityHouses =
+                communityApiMapper.communityHouseDtoSetToCommunityHouseSet(request.getHouses());
+        var houseIds = communityService.addHousesToCommunity(communityId, communityHouses);
+        var response = new AddCommunityHouseResponse();
+        response.setHouses(houseIds);
+        return ResponseEntity.status(HttpStatus.CREATED).body(response);
+    }
+
+
+    @Operation(description = "Delete a house from the community given a community id and a house id")
+    @DeleteMapping(
+            path = "/communities/{communityId}/deleteHouse/{houseId}",
```

Comment:
Following REST API best practices. Can the endpoint be `/communities/{communityId}/houses/{houseId}`?

Reply:
Changed ;)